addons/account/views/account_move_views.xml

```diff
@@ -264,7 +250,10 @@
             <field name="mode">primary</field>
             <field name="inherit_id" ref="account.view_move_line_tree_grouped"/>
             <field name="arch" type="xml">
-                <field name="account_id" position="attributes"><attribute name="optional">hide</attribute></field>
+                <field name="account_id" position="attributes">
+                    <attribute name="optional">hide</attribute>
+                    <attribute name="readonly">1</attribute>
+                </field>
```

Comment:
this is to remove as well

Reply:
No because the move lines are grouped by account on the general ledger view